util/activity-report/test/avg-tx-cost.e2e.test.js

```diff
@@ -0,0 +1,106 @@
+const { test, request, wait } = require('./util.js');
+
+const url = '/api/v1/rsk-activity-report/avg-tx-cost';
+const chain = 'rsk_mainnet';
+const blocks = 100;
+
+/**
+ * Success
+ */
+
+// first request to the API should return a response body with an empty object {}
+// this is because nothing is cached yet
+test('Average tx cost. The first request should return {} in res.body', async (assert) => {
+  const res = await request.get(url).query({ chain, blocks });
+  assert.equal(res.status, 200, 'response status should be 200');
+  assert.deepEqual(res.body, {}, 'res.body should be {}');
+  assert.end();
+});
```

Comment:
we might to revisit the implementation here (for another PR). just bookmarking this for now, let's discuss later.